workflow/rules/generate_output.smk

```diff
@@ -129,7 +115,7 @@ rule virologist_report:
         ),
         initial_contigs=lambda wildcards: get_expanded_contigs(wildcards),
         polished_contigs=lambda wildcards: expand(
-            "results/{{date}}/polished-contigs/{sample}.fasta",
+            "results/{{date}}/contigs/polished/{sample}.fasta",
```

Comment:
Mhm, here I am confused. shouldn't this either obtain the polished (or better even masked) or the pseudoassembled contigs, depending on the rki filtering?

Reply:
I actually wasn't sure, how we should handle this. The current use of this input is the calculation of the `Final Contig (bp)` length, thus it provides information about the "normal assembly". I didn't want to skew our view on this data by mixing it with information from the pseudo assembly. 

I thought about adding another column with the length of the pseudo assembly. But what is the added value for the virologist, when would do this?

However, I would love to hear your opinions about this.

Reply:
Ah, I see. I think it is best to have one column for the assembly length, one for the pseudoassembly length and one denoting which assembly was chosen.

Reply:
Added this in 009a33a :)